### PR TITLE
GSoC: Adding Week 4 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
image

### CODE REVIEW

**Review by tvcastillod — COMMENTED** [review]
Hi, @ganimtron-10, it looks like it was a fruitful week despite being busy with exams, great job. The post was generated correctly, so everything looks good to me.

File: docs/source/posts/2023/2023-06-24-week-4-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+1. Merging **CardUI**: The PR `#398 <https://github.com/fury-gl/fury/pull/398>`_ introduced the **CardUI** to the UI system of FURY. After a successful review and test check, it was merged into the codebase.
+
+2. Revisiting PR `#540 <https://github.com/fury-gl/fury/pull/540>`_: I restarted working on PR `#540 <https://github.com/fury-gl/fury/pull/540>`_ as I wasn't satisfied with the previous approach when I checked it for rebasing. I took the opportunity to update the code and ensure that the unit tests passed successfully. Although there are a few issues remaining in the tests, I am determined to resolve them and move forward with the implementation. This PR aims to improve the usage of the **numpy_to_vtk_image_data** utility function.
+
+3. Independent Scrollbar Consideration: We are currently evaluating the necessity of making the Scrollbar an independent element. While currently, it is primarily used by the **ListBox2D**, we are exploring various use cases to determine if there are other scenarios where the Scrollbar can be employed independently. This evaluation will help us make an informed decision about its future implementation.
```
Comment (JoaoDell) [inline docs/source/posts/2023/2023-06-24-week-4-praneeth.rst:18]: Here I would just take the comma: `While currently, it is primarily` -> `While currently it is primarily`

**Review by JoaoDell — CHANGES_REQUESTED** [review]
Well done Praneeth, I am amidst end-semester exams as well, so seeing you accomplish so many things while going through that is impressive, good luck with all of that! About your blogpost, everything seems fine, just one change I would do below

**Comment by ganimtron-10** [comment]
Thanks for the review @tvcastillod & @JoaoDell.

**Review by skoudoro — APPROVED** [review]
All good, Thanks to the reviewers and @ganimtron-10.

Merging

**Pull request merged**